sdk/state/close.go

```diff
@@ -26,6 +26,16 @@ func (cc CoordinatedClose) CloseSignatures() []xdr.DecoratedSignature {
 	return cc.closeSignatures
 }
 
+// mergeCoordinatedCloseData merges the data from a new coordinated close into an existing one. The signatures of the existing
+// coordinated close are appended to so that existing signatures are not lost.
+func mergeCoordinatedCloseData(cc CoordinatedClose, newCoordinatedClose CoordinatedClose) CoordinatedClose {
```

Comment:
I'm not tied to this naming using "merge". Some of the data fields are overwritten, and then the signatures are the ones actually merged so none are lost. But I couldn't think of a better name for the method

Reply:
I think what you highlight isn't an issue with the function name but the function responsibilities. I think we should call `appendNewSignatures` directly.

Reply:
removed the merge methods here: https://github.com/stellar/experimental-payment-channels/pull/88/commits/ec2b6306997dc887ef74c30809a0a1c40d822b5a

definitely makes it cleaner since we removed the observation fields from the `CoordinatedClosed` struct